CHANGELOG.md

```diff
@@ -5,7 +5,16 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 
 
-##  Hammer Beta-2
+## Hammer RC-1
+
+### Added
+- Deny request if no conversion host is configured [(#455)](https://github.com/ManageIQ/manageiq-content/pull/455)
+- Use task cancelation_status to trigger cancelation [(#457)](https://github.com/ManageIQ/manageiq-content/pull/457)
+
+### Fixed
+- Add manageiq automate changes [(#454)](https://github.com/ManageIQ/manageiq-content/pull/454)
```

Comment:
This PR title was non-descriptive.  I have updated the PR to read "Apply updates to the manageiq-automate Ansible role" and we should do the same here.  Thanks.